Add links to list of users of the Standard (#120)
publiccodenet/community-implementation-guide-standard: index.md

We should keep in mind "there's more than one way to do it": where it makes sense, provide multiple options, especially linking to examples.

> For inspiration of existing solutions that may not have been added here yet, check the assessments of [users of the Standard for Public Code](https://github.com/standard-for-public-code#whos-using-the-standard-for-public-code).

Reviewer: Alternative wording suggestion, since this page is more conversational.

```suggestion
For further inspiration, check out [existing users' assessments against the Standard for Public Code](https://github.com/standard-for-public-code#whos-using-the-standard-for-public-code).
```
